roboteam_ai/src/skills/Dribble.h

```diff
@@ -19,6 +19,7 @@ class Dribble : public Skill {
     private:
         enum Progression {
           ON_THE_WAY,
+          STOPPING,
```

Comment:
I'd rather see this removed again, this progression state is unnecessary as it executes the same commands as the stopped state.

Reply:
Yes, but multiple times if requested, sometimes 1 tick is not enough to properly lay the ball still

Reply:
you can adjust the amount of ticks to lay the ball still in constants or in the skill in the tree. This is still duplicate functionality.